Add a summary table to therapeutics (#865)
greenelab/covid19-review: content/20.pharmaceuticals.md

#### Conclusions <!--I’m not sure we need this section. We’ve stated these things before in this manuscript and the section reads as how useful it’s been to learn from SARS-CoV-2 and MERS-CoV – it has been useful, but I don’t think we need that as the overall conclusion…-->

| Treatment | Category | FDA Status | Evidence Available | Suggested Effectiveness |
|-----------|----------|-----------------|--------------------|-------------------------|
| Favipiravir | Small molecule, antiviral, nucleoside analog | None | RCT | Not supported | 
| Remdesivir | Small molecule, antiviral, adenosine analog | Approved (and EUA for combination with baricitinib) | RCT | Conflicting evidence from Solidarity vs other trials |
| N3 | Small molecule, protease inhibitor | None | Computational prediction | Unknown |

> | ARBs & ACEIs | Small molecule, broad spectrum | None | RCT | Not supported |

Author: ```suggestion
| ARBs & ACEIs | Small molecule, broad spectrum | None | Observational studies and some RCTs | Not supported: Observational study retracted, RCTs suggest no association |
```

> | HCQ/CQ | Small molecule, broad spectrum | None | RCT  | Not supported, possibly harmful |

Author: ```suggestion
| HCQ/CQ | Small molecule, broad spectrum | None | RCT  | Not supported, possibly harmful: Non-blinded RCTs showed no improvement over SOC, safety profile may be problematic |
```

> | Dexamethasone | Small molecule, broad spectrum | Used off-label | RCT | Supported |

Author: ```suggestion
| Dexamethasone | Small molecule, broad spectrum | Used off-label | RCT | Supported: RCT shows improved outcomes over SOC, especially in severe cases such as CRS |
```

> | Tocilizumab | Biologic, monoclonal antibody | Approved for CRS | RCT | Supported |

Author: ```suggestion
| Tocilizumab | Biologic, monoclonal antibody | Approved for CRS resulting from CAR-T therapy | RCT | Mixed results from RCTs: It appears that TCZ may work well in combination with dexamethasone in severe cases, but not as monotherapy |
```

| Possibly other antibodies? | 
| Casirivimab and imdevimab | Biologic, monoclonal antibodies | EUA | | | |
| Bamlanivimab and etesevimab | Biologic, monoclonal antibodies | EUA | | | |

> | SNG001 | Biologic, interferon | None | RCT | Likely not supported |

Author: ```suggestion
| SNG001 | Biologic, interferon | None | RCT | Mixed results: support from initial RCT but no effect found in WHO's Solidarity trial |
```
